cypress/e2e/tests/migration/applicationinventory/analysis/analysis_with_proxyenabled.test.ts

```
@@ -0,0 +1,63 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import { getRandomProxyData } from "../../../../../utils/data_utils";
+import { getRandomAnalysisData, getRandomApplicationData } from "../../../../../utils/utils";
+import { Proxy } from "../../../../models/administration/proxy/proxy";
```

Biome (2.1.2)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)